src/api.js

```diff
@@ -0,0 +1,139 @@
+"use strict";
+
+const RECORD_FIELDS_TO_CLEAN = ["_status", "last_modified"];
+
+export function cleanRecord(record, excludeFields=RECORD_FIELDS_TO_CLEAN) {
+  return Object.keys(record).reduce((acc, key) => {
+    if (excludeFields.indexOf(key) === -1)
+      acc[key] = record[key];
+    return acc;
+  }, {});
+};
+
+const DEFAULT_REQUEST_HEADERS = {
+  "Accept":       "application/json",
+  "Content-Type": "application/json",
+};
```

Comment:
Note for later:
``# XXX: This could probably be an attribute of the Api class, so that developers can get a hand on it to add their own headers.``